packs/stolen-fate-bestiary/blood-pear-tree.json

```diff
@@ -110,17 +110,19 @@
                     "0": {
                         "damage": "2d6+12",
                         "damageType": "bludgeoning"
-                    },
-                    "1": {
-                        "category": "splash",
-                        "damage": "2d6",
-                        "damageType": "poison"
                     }
                 },
                 "description": {
-                    "value": ""
+                    "value": "<p>Plus [[/r (2d6[splash])[bleed]]]{2d6 bleed splash damage}</p>"
```

Comment:
Bleed splash damage is certainly nonstandard, and I don't expect we should support it. Does not work for damageRolls or DamageDice.
However, it does work for inline rolls. If this is considered too fragile, I'm happy to change the inline roll to regular bleed.